server/src/main/java/com/hedera/block/server/persistence/storage/PersistenceStorageConfig.java

```diff
@@ -43,7 +47,9 @@ public record PersistenceStorageConfig(
         @ConfigProperty(defaultValue = "") String liveRootPath,
         // @todo(#371) - the default life/archive root path must be absolute starting from /opt
         @ConfigProperty(defaultValue = "") String archiveRootPath,
-        @ConfigProperty(defaultValue = "BLOCK_AS_LOCAL_FILE") StorageType type) {
+        @ConfigProperty(defaultValue = "BLOCK_AS_LOCAL_FILE") StorageType type,
+        @ConfigProperty(defaultValue = "ZSTD") CompressionType compression,
+        @ConfigProperty(defaultValue = "3") int compressionLevel) {
```

Comment:
I believe we should set bounds for this config.
Also, while zstd allows for negative value, I believe we should constrain to strictly greater than or equal to `0`.
```suggestion
        @ConfigProperty(defaultValue = "3") @Min(0) @Max(20) int compressionLevel) {
```

Reply:
✅ Done.